### PR TITLE
Adding bootleg and waffling60 rev b

### PR DESCRIPTION
**Pull request by 4pplet**
## Description

Adding bootleg (non tray-mount 60% PCB) and waffling60 rev b (Updated matrix and RGB-led implementation that fits QMK better). Thanks! :)

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by drashna — APPROVED** [review]

**Comment by drashna** [comment]
Looks like you have a merge conflict that needs resolving.

**Comment by 4pplet** [comment]
Thanks for reviewing! Conflict should be solved now, also added readmes in revision folders so that lint-checks don't fail.

**Review by drashna — APPROVED** [review]

File: keyboards/4pplet/bootleg/README.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# bootleg
```
Comment (fauxpark) [inline keyboards/4pplet/bootleg/README.md:1]: Please rename this file to lowercase.

File: keyboards/4pplet/bootleg/rev_a/config.h [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x4444
+#define PRODUCT_ID      0x0004
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    4pplet
+#define PRODUCT         bootleg Rev A
+#define DESCRIPTION
```
Comment (fauxpark) [inline keyboards/4pplet/bootleg/rev_a/config.h:27]:
```suggestion
```

File: keyboards/4pplet/bootleg/rev_a/rules.mk [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+MCU = atmega32u2
+
+# Bootloader selection
+#   Teensy       halfkay
+#   Pro Micro    caterina
+#   Atmel DFU    atmel-dfu
+#   LUFA DFU     lufa-dfu
+#   QMK DFU      qmk-dfu
+#   ATmega32A    bootloadHID
+#   ATmega328P   USBasp
```
Comment (fauxpark) [inline keyboards/4pplet/bootleg/rev_a/rules.mk:11]:
```suggestion
```

File: keyboards/4pplet/bootleg/rev_a/rules.mk [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = no            # USB Nkey Rollover 
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+MIDI_ENABLE = no            # MIDI support
+AUDIO_ENABLE = no           # Audio output on port C6
+FAUXCLICKY_ENABLE = no      # Use buzzer to emulate clicky switches
+HD44780_ENABLE = no         # Enable support for HD44780 based LCDs
+UNICODE_ENABLE = no         # Unicode
```
Comment (fauxpark) [inline keyboards/4pplet/bootleg/rev_a/rules.mk:32]:
```suggestion
BLUETOOTH_ENABLE = no       # Enable Bluetooth
AUDIO_ENABLE = no           # Audio output
```

File: keyboards/4pplet/waffling60/rev_b/config.h [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x4444
+#define PRODUCT_ID      0x0005
+#define DEVICE_VER      0x0002
+#define MANUFACTURER    4pplet
+#define PRODUCT         waffling60 Rev B
+#define DESCRIPTION     4pplet Keyboards
```
Comment (fauxpark) [inline keyboards/4pplet/waffling60/rev_b/config.h:27]:
```suggestion
```

File: keyboards/4pplet/waffling60/rev_b/rev_b.c [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+*/
+#include "rev_b.h"
+
+void matrix_init_kb(void) {
+    // put your keyboard start-up code here
+    // runs once when the firmware starts up
+
+    matrix_init_user();
+}
+
```
Comment (fauxpark) [inline keyboards/4pplet/waffling60/rev_b/rev_b.c:25]:
```suggestion
```

File: keyboards/4pplet/waffling60/rev_b/rules.mk [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+MCU = atmega32u2
+
+# Bootloader selection
+#   Teensy       halfkay
+#   Pro Micro    caterina
+#   Atmel DFU    atmel-dfu
+#   LUFA DFU     lufa-dfu
+#   QMK DFU      qmk-dfu
+#   ATmega32A    bootloadHID
+#   ATmega328P   USBasp
```
Comment (fauxpark) [inline keyboards/4pplet/waffling60/rev_b/rules.mk:11]:
```suggestion
```

File: keyboards/4pplet/waffling60/rev_b/rules.mk [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = no            # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = yes        # Enable keyboard RGB underglow
+MIDI_ENABLE = no            # MIDI support
+AUDIO_ENABLE = no           # Audio output on port C6
+FAUXCLICKY_ENABLE = no      # Use buzzer to emulate clicky switches
+UNICODE_ENABLE = no         # Unicode
```
Comment (fauxpark) [inline keyboards/4pplet/waffling60/rev_b/rules.mk:31]:
```suggestion
RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
BLUETOOTH_ENABLE            # Enable Bluetooth
AUDIO_ENABLE = no           # Audio output
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/4pplet/bootleg/README.md (hunk 1 above)
Comment (4pplet) [reply keyboards/4pplet/bootleg/README.md]: Thanks for the feedback and changes! Filename is changed to lower case.

**Review by fauxpark — APPROVED** [review]

**Comment by 4pplet** [comment]
Everything should be in order for merge? :)

**Comment by fauxpark** [comment]
Thanks!

**Pull request merged**